routes/company.php

```diff
@@ -6,4 +6,10 @@
     Route::post('register','register') ;
     Route::post('login','login');
     Route::get('logout','logout')->middleware('auth:company');
+    Route::get('refresh','refresh')->middleware('auth:company');
+});
+
+Route::controller(\App\Http\Controllers\company\JobController::class)->group(function() {
```

Comment:
add middleware to the group 
`Route::controller(\App\Http\Controllers\company\JobController::class)->middleware('auth:company')->group(function() {`